src/libs/OptionsListUtils.js

```diff
@@ -87,6 +87,7 @@ Onyx.connect({
 const policyExpenseReports = {};
 Onyx.connect({
     key: ONYXKEYS.COLLECTION.REPORT,
+    waitForCollectionCallback: true,
```

Comment:
This caused serious regression. App crashes when request expense


https://github.com/Expensify/App/assets/108292595/7f00f5c5-1aea-4d3f-8420-0b61906d345b



Anyone please raise quick fix?

Reply:
I just encountered this issue too. Pushing a fix soon

Reply:
@Gonals being reverted in https://github.com/Expensify/App/pull/30852